flow/layers/shader_mask_layer.h

```diff
@@ -27,6 +27,9 @@ class ShaderMaskLayer : public ContainerLayer {
   SkRect mask_rect_;
   SkBlendMode blend_mode_;
 
+  static constexpr int kMinimumRendersBeforeCachingFilterLayer = 3;
+  int render_count_;
```

Comment:
Please initialize `render_count_` in constructor just like other "cacheable" layers.